### PR TITLE
Remove dead assignments in RecoLocalMuon.CSCValidation.CSCValidation

### PR DESCRIPTION
**Pull request by perrotta**
#### PR description:

There were a few "Dead assignment" issues that popped out from the static analyzer in CSCValidation.
This PR intends to fix them (and apply also some further little cleaning)

#### PR validation:

It compiles
No changes expected

### CODE REVIEW

File: RecoLocalMuon/CSCValidation/src/CSCValidation.cc [hunk 1]
```diff
@@ -859,18 +860,10 @@ void CSCValidation::doPedestalNoise(edm::Handle<CSCStripDigiCollection> strips)
       int myStrip = pedIt->getStrip();
       std::vector<int> myADCVals = pedIt->getADCCounts();
       float TotalADC = getSignal(*strips, id, myStrip);
-      bool thisStripFired = false;
       float thisPedestal = 0.5 * (float)(myADCVals[0] + myADCVals[1]);
       float thisSignal =
           (1. / 6) * (myADCVals[2] + myADCVals[3] + myADCVals[4] + myADCVals[5] + myADCVals[6] + myADCVals[7]);
-      float threshold = 13.3;
-      if (id.station() == 1 && id.ring() == 4) {
```
Comment (perrotta) [inline RecoLocalMuon/CSCValidation/src/CSCValidation.cc:867]: @ptcox I'm puzzled about this dead reassignment of `myStrip`: was it intended to be somewhere else (e.g. above)?

**Comment by perrotta** [comment]
please test

**Comment by ptcox** [comment]
Hi Andrea,

Thanks for drawing my attention to this.
It's completely beyond my comprehension too :)

However, it is pretty clear what it was once trying to do: in run 1 we 
had the 48 strips of ME1/1A ganged every 3rd strip, so the 48 strips fed 
into 16 channels.
In the offline, strip channels in ME1/1A were counted 1-16. (And now 
they count 1-48).
But in the hardware, the count was tacked on after the 64 channels of 
ME1/1B - so the software/real channel count 1-16 <-> a hardware/online 
channel count 65-80. (This made some sense because in all other chamber 
there are 80 channels, and so the count 1-80 is 'natural').

But WHY the code contains a reminder of this is beyond me. I presume it 
was something introduced at some long ago time in the past before the 
Unpacker decoded channels properly.

So I think there is no issues in suppressing it.

But thanks for noticing.

Regards, Tim

**Comment by perrotta** [comment]
Thank you @ptcox for your answer!

**Comment by perrotta** [comment]
+1
- Dead assignments removed from RecoLocalMuon CSCValidation code 
- This has no effect on the flow of the code, and jenkins tests show no differences in output

**Comment by qliphy** [comment]
+1

**Pull request merged**